tests/drivers/pwm/pwm_loopback/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+source "Kconfig.zephyr"
+
+config SKIP_PWM_INVERTED_TEST
```

Comment:
This should not be a Kconfig. Instead, the test suite should attempt to configure the test for inverted polarity, but skip it using `ztest_test_skip()` if inverted polarity is not supported.